kolibri_instant_schools_plugin/assets/src/views/reset-password-modal/index.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+
+  <core-modal
+    :title="currentTitle"
+    @cancel="closeModal"
+  >
+    <div class="contents">
+      <status
```

Comment:
Instead of having a modal for `Account not found` it might be better if we just use a `ui-alert`. Also it does not help that when you click back, the field is empty, so you cannot see the inputted phone number.